test/scenarios/re_emissions_elec_only.json

```diff
@@ -48,7 +48,7 @@
     "production_incentive_max_benefit": 0.0,
     "state_ibi_fraction": 0.0,
     "electric_efficiency_full_load": 0.349006,
-    "electric_efficiency_half_load": 0.326729,
```

Comment:
@zolanaj just curious why you made this change?

Reply:
by making the part-load and full-load efficiency the same, we no longer have fuel burn tied to the binary variables indicating that the technology is running in a given hour.  This effectively make the fuel burn constraints linear.